src/components/Web3Status/ConnectWalletPopover.tsx

```diff
@@ -11,23 +11,49 @@ import Popover from '../Popover'
 import { useCloseModals, useModalOpen } from '../../state/application/hooks'
 import { ApplicationModal } from '../../state/application/actions'
 import { useOnClickOutside } from '../../hooks/useOnClickOutside'
+import connectedSvg from '../../assets/svg/connected.svg'
 
 const Wrapper = styled.div`
   width: 100%;
 `
 
+const ConnectedIcon = styled.img`
+  margin-left: 6px;
+`
```

Comment:
![Screenshot 2021-12-20 at 19 05 18](https://user-images.githubusercontent.com/33226956/146812585-df341241-1c75-45ad-ba14-1adbcd791bae.png)
Is there some better solution then just manually aligning it like this since its not 100% aligned?